elements/pl-code/pl-code.py

```diff
@@ -21,6 +24,22 @@
 COPY_CODE_BUTTON_DEFAULT = False
 
 
+class AnsiColorStyle(Style):
+    styles = dict(get_style_by_name("friendly").styles)
+    # These are the same colors used in pl-external-grader-result
+    ansi_colors = {
```

Comment:
If these colors are duplicated, might be a good idea to put these in a shared file somewhere?

Reply:
Yes, I thought about that. There is a minor issue that ansi2html expects an ordered array while pygments requires a dict, so there is a need to adjust from one to the other, though this is doable.

Another option is to have pl-external-grader-result defer to pl-code for this functionality, so all colors would be found here.

Reply:
Having external grader result defer is probably the best option imo, since it should mean getting to simplify the backend of that element, similar to what we did for pl-variable-output.

Reply:
Unfortunately `pygments-ansi-color` doesn't currently support bright colors (https://github.com/chriskuehl/pygments-ansi-color/issues/8). So we should either keep both elements separate, or lose access to bright colors in pl-external-grader-result.

Reply:
Submitted https://github.com/chriskuehl/pygments-ansi-color/pull/27, if it is pushed and deployed relatively soon we can use it.

Reply:
The update was merged and deployed to pip, so 8eb0187ad updates the code to use bright colors as well.

Reply:
Moved this discussion to #7191.